hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/TxnVerbs.java

```diff
@@ -652,14 +653,28 @@ public static HapiContractUpdate contractUpdate(String contract) {
      * @param contractsNames the name(s) of the contract(s), which are to be deployed
      */
     public static HapiSpecOperation uploadInitCode(final String... contractsNames) {
-        return uploadInitCode(Optional.empty(), contractsNames);
+        return uploadInitCode("", Optional.empty(), contractsNames);
     }
 
-    public static HapiSpecOperation uploadInitCode(final Optional<String> payer, final String... contractsNames) {
+    public static HapiSpecOperation uploadInitCode(
```

Comment:
So we've got two overloads of this method that are distinguished by the first two parameters being in one order or the other ... this is odd ... why?

Reply:
The zero or more String parameters for the contract names makes things a little tricky.  I think this overload does not make sense.  I will remove it.